src/Bonsai.ML.Torch/ValueTupleConverter.cs

```diff
@@ -0,0 +1,496 @@
+using System;
+using System.ComponentModel;
+
+namespace Bonsai.ML.Torch;
+
+/// <summary>
+/// Type converter for single-element value tuples.
+/// </summary>
+/// <typeparam name="T"></typeparam>
```

Comment:
Type params can also be documented to indicate what they refer to, for example in this case we could say something like "The type of the single-element in the value tuple."